sdk/src/main/java/com/mapbox/maps/renderer/widget/BitmapWidgetRenderer.kt

```diff
@@ -246,6 +247,7 @@ internal class BitmapWidgetRenderer(
       GLUtils.texImage2D(GLES20.GL_TEXTURE_2D, 0, it, 0)
       GLES20.glBindTexture(GLES20.GL_TEXTURE_2D, 0)
 
+      onBitmapLoaded(it)
```

Comment:
Could we add a test to check this is called? (I guess it needs to be a instrumentation test)
And it will most likely only work when building the widget for first time. `updateBitmap` won't trigger it since we don't have yet the force refresh.